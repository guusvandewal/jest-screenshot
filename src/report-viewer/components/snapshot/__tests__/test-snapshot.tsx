import * as React from "react";
import { shallow } from "enzyme";
import { Snapshot } from "..";

describe("Snapshot", () => {
    it("looks as expected", () => {
        expect(shallow(
            <Snapshot
                snapshot={testResults.files[0].failedTests[0].failedSnapshots[0]}
            />,
        )).toMatchSnapshot();
    });
});
