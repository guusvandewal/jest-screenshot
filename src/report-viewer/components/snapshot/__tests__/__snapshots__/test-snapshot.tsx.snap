// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot looks as expected 1`] = `
<article>
  <nav
    className="level"
  >
    <LevelItem
      name="Total Pixels"
      value="10"
    />
    <LevelItem
      name="Changed Pixels"
      value="5"
    />
    <LevelItem
      name="Difference"
      value="50.00%"
    />
  </nav>
  <div
    className="message test-message"
  >
    <div
      className="message-body"
    >
      <Ansi>
        Received message.
      </Ansi>
    </div>
  </div>
  <ImageDiffViewer
    diff="jest-screenshot-report/reports/some-identifier.snap.png/diff.png"
    height={2}
    received="jest-screenshot-report/reports/some-identifier.snap.png/received.png"
    snapshot="jest-screenshot-report/reports/some-identifier.snap.png/snapshot.png"
    width={5}
  />
</article>
`;
